beacon-chain/forkchoice/doubly-linked-tree/proposer_boost.go

```diff
@@ -64,9 +37,7 @@ func computeProposerBoostScore(validatorBalances []uint64) (score uint64, err er
 		err = errors.New("no active validators")
 		return
 	}
-	avgBalance := totalActiveBalance / numActive
-	committeeSize := numActive / uint64(params.BeaconConfig().SlotsPerEpoch)
-	committeeWeight := committeeSize * avgBalance
+	committeeWeight := totalActiveBalance / uint64(params.BeaconConfig().SlotsPerEpoch)
```

Comment:
note to ask Raul, I don't know if there was an overflow concern to have a common factor being out